darwin/exporter/formats/nifti.py

```diff
@@ -97,7 +101,9 @@ def build_output_volumes(video_annotation: dt.AnnotationFile) -> Dict:
     for slot in video_annotation.slots:
         slot_metadata = slot.metadata
         assert slot_metadata is not None
-        series_instance_uid = slot_metadata.get("SeriesInstanceUID", "SeriesIntanceUIDNotProvided")
+        series_instance_uid = slot_metadata.get(
+            "SeriesInstanceUID", "SeriesIntanceUIDNotProvided"
```

Comment:
We don't want to remove these two then in the end? Or did you split it into a different PR?